packages/node/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "readmeio",
   "description": "Track your API metrics within ReadMe.",
-  "version": "4.2.0",
+  "version": "5.0.0",
```

Comment:
Undo this change, the Lerna publish process will take care of it automatically

Reply:
Even if it's a significant breaking change and needs a major version update?

Reply:
Yeah when you run `npm run publish` from the root Lerna will ask you what version to release it as and then automatically generate a changelog. Only works for Node since JS is all Lerna supports though.

Reply:
👍